src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/PointsToAnalysis/TrackedEntitiesBuilder.cs

```diff
@@ -16,43 +16,43 @@ internal sealed class TrackedEntitiesBuilder : IDisposable
         /// Stores all the tracked entities.
         /// NOTE: Entities added to this set should not be removed.
         /// </summary>
-        private PooledHashSet<AnalysisEntity> _allEntities { get; }
+        private PooledHashSet<AnalysisEntity> AllEntities { get; }
```

Comment:
Is it worth having it as a property?

Reply:
Since these were already properties, it seems out of scope for this PR.